### ISSUE TITLE
File Explorer does not show the coverage bar

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#36024]
Does this issue occur when all extensions are disabled?: Yes/No

Version: 1.86.0-insider
Commit: 46a233646602c378c2168fa2a104dff8e8551558
Date: 2023-12-11T05:36:45.890Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Code-Insiders/1.86.0-insider Chrome/118.0.5993.159 Electron/27.1.3 Safari/537.36

Hi -connor4312 

Sorry I cannot reproduce this issue via the extension sample.

Basically, I feed the test coverage provider with the file coverages. And the test coverage view shows them. And I'm sure that `testing.showCoverageInExplorer` is enabled.

What could be the reason causing that?

![image](https://github-com/microsoft/vscode/assets/6193897/6a95ef14-cad1-4f9f-b8ac-dfb355442f9a)
